src/Shared/Recruit.Vacancies.Client/Domain/Entities/ProgrammeLevelHelper.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+
+namespace Esfa.Recruit.Vacancies.Client.Domain.Entities
+{
+    public static class ProgrammeLevelHelper
+    {
+        public static bool TryRemapFromInt(int value, out ProgrammeLevel result)
```

Comment:
Can this method please be static as it is only being called from one place which is within this class.

Reply:
Sorry meant `private static`

Reply:
I have some tests for it, so I'll leave it public